pkg/capabilities/consensus/ocr3/aggregators/reduce_aggregator.go

```diff
@@ -185,6 +182,53 @@ func (a *reduceAggregator) Aggregate(lggr logger.Logger, previousOutcome *types.
 	}, nil
 }
 
+func (a *reduceAggregator) shouldReport(lggr logger.Logger, field AggregationField, singleValue values.Value, currentState *map[string]values.Value) (bool, error) {
+	oldValue := (*currentState)[field.OutputKey]
+
+	// this means its the first round and the field has not been initialised
+	if oldValue == nil {
+		return true, nil
+	}
+
+	if field.DeviationType == DEVIATION_TYPE_NONE {
+		unwrappedOldValue, err := oldValue.Unwrap()
+		if err != nil {
+			return false, err
+		}
+
+		unwrappedSingleValue, err := singleValue.Unwrap()
+		if err != nil {
+			return false, err
+		}
+
+		// we will only report in case of a change in value
+		switch v := unwrappedOldValue.(type) {
+		case []byte:
+			if !bytes.Equal(v, unwrappedSingleValue.([]byte)) {
+				return true, nil
+			}
+		default:
+			if unwrappedOldValue != unwrappedSingleValue {
+				return true, nil
+			}
+		}
```

Comment:
🤔 for `DEVIATION_TYPE_NONE` Im considering field types as `[]byte` or `bool` are there any other type I need to check?